xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/hierarchy_macros.vm

```diff
@@ -146,7 +146,8 @@
       #set ($breadCrumbParameters = "$breadCrumbParameters data-${option.toLowerCase()}='$escapetool.xml($options.get($option))'")
     #end
   #end
-  <ol #if("$!options.id" != '')id="$options.id"#end class="$breadCrumbClassNames" $breadCrumbParameters><li
+  <ol #if("$!options.id" != '')id="$options.id"#end class="$breadCrumbClassNames" $breadCrumbParameters>
+  <li
```

Comment:
I think this line break should be removed again, too.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2133/commits/551cce136d80c1ba8703aa789488c1450d0d79cf :+1: